integrations/sharepoint/src/index.ts

```diff
@@ -0,0 +1,111 @@
+import * as sdk from "@botpress/sdk";
+import * as bp from ".botpress";
+import { BotpressKB } from "./BotpressKB";
+import { createSharepointClient } from "./SharepointClient";
+import { log } from "console";
+
+export default new bp.Integration({
+  register: async ({ ctx, webhookUrl, client, logger }) => {
+    try {
+      const spClient = createSharepointClient(
+        ctx.configuration,
+        new BotpressKB(client, ctx.configuration.kbId, logger)
+      );
+
+      logger.forBot().info(`[Registeration]: Registering webhook with URL: ${webhookUrl}`);
```

Comment:
**nitpick:** typo in word `registration` :)

Reply:
fixed!